annotation-processor/src/main/java/org/hibernate/validator/ap/ConstraintAnnotationVisitor.java

```diff
@@ -6,9 +6,11 @@
  */
 package org.hibernate.validator.ap;
 
+import java.util.Collections;
 import java.util.List;
+import java.util.Map;
 import java.util.Set;
-
+import java.util.stream.Collectors;
```

Comment:
Well I kind of like this stream API and lambdas, but i suppose I can change it all to regular loops. And maybe later, when the time for 6.0 comes) create a task to update it all back to Java 8 stuff :)

As a follow up question - in a meanwhile I'm working on HV-840 - it's a part of annotation processor as well, so I'd better not use lambdas/streams there as well right ?